src/Entities/Transaction.ts

```diff
@@ -58,6 +61,18 @@ export class Transaction {
     return transaction;
   }
 
+  // TODO see if we actually need this
+  public static fromClientId(
+    clientTransactionId: string,
+    paymentMethodType = PaymentMethodType.Credit,
+  ) {
```

Comment:
Consider adding the return type so that it doesn't have to be inferred

Reply:
I don't think we're going to need this block of code, i forgot to remove it.